package.json

```diff
@@ -51,7 +51,7 @@
     "@dapplion/benchmark": "^0.2.4",
     "@biomejs/biome": "^1.9.3",
     "@types/mocha": "^10.0.6",
-    "@types/node": "^20.12.8",
+    "@types/node": "^22.7.5",
```

Comment:
this should only be updated if we plan to drop node 20 support

Reply:
Was not intended for this PR, was checking some type issue.